packages/components/label/tests/label.test.ts

```diff
@@ -44,17 +48,16 @@ describe.skip('okuLabel', () => {
     expect(await axe(wrapper.element)).toHaveNoViolations()
   })
 
+  // TODO: mock preventDefault()
   it('should prevent default on double click', async () => {
-    const event = {
-      preventDefault: vi.fn(),
-      detail: 2,
-    }
-
-    await wrapper.trigger('mousedown', event)
+    await wrapper.trigger('mousedown')
+    await wrapper.trigger('mousedown')
 
     expect(wrapper.emitted('mousedown')).toBeTruthy()
+  })
```

Comment:
@productdevbook Should we remove this test, i could not mock the preventDefault().

Reply:
leave it for now, :+1: